swagger/swagger.yml

```diff
@@ -0,0 +1,68 @@
+consumes:
+- application/json
+definitions:
+  account:
```

Comment:
where is `id`?
that represents account_id.

Reply:
Now added. Please check